src/generator/operations.ts

```diff
@@ -947,6 +947,30 @@ function createProtocolRequest(group: OperationGroup, op: Operation, imports: Im
     text += '\t\treturn nil, err\n';
     text += '\t}\n';
     text += '\treturn req, nil\n';
+  } else if (mediaType === 'form') {
```

Comment:
Can this be merged with the `multipart` section or does it have to be separate?

Reply:
It has to be separate. According to the w3 [spec](https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4), the body encoding method for `application/x-www-form-urlencoded` and `multipart/form-data are different.